features/gba-dev/europees-kiesrecht-gba.feature

```diff
@@ -0,0 +1,62 @@
+# language: nl
+
+Functionaliteit: Europees kiesrecht
+
+  Scenario: Geen Europees kiesrecht opgegevens, geen Europees kiesrecht (31.10) en geen einddatum uitsluiting Europees kiesrecht (31.30)
```

Comment:
Zou het graag even voor je wijzigen, maar weet eigenlijk niet goed wat hier moet staan :-). Geen Europees kiesrecht op gegevens of geen Europees kiesrecht opgegeven? Of kan het eerste deel van de zin gewoon weg, omdat het geen toegevoegde waarde heeft voor het deel achter de komma?

Reply:
Aangepast (eerste deel van de zin weggehaald. Is duidelijker.